tests/httpwrappers/tests.py

```diff
@@ -531,6 +531,50 @@ def test_content_type_mutually_exclusive(self):
             )
 
 
+def test_text_property(self):
```

Comment:
This suite is unittest-style, not pytest-style, so this test needs to be enclosed in a class for it to run.

Reply:
I intended to implement the method as an instance method within the class, but I accidentally defined it as a standalone function. Thank you for the guidance